### PR TITLE
RHDEVDOCS-4985: Updated the matrix table to fill in identified gaps

### PR DESCRIPTION
**Pull request by shipsing**
Aligned team: Dev Tools

Purpose: To resolve the following issues:
https://issues.redhat.com/browse/RHDEVDOCS-4985

OCP version this PR applies to: enterprise 4.9 and later

Link to docs preview: https://56355--docspreview.netlify.app/openshift-enterprise/latest/cicd/pipelines/op-release-notes.html#compatibility-support-matrix_op-release-notes

Change Management
Need an ACK from Eng: @savitaashture @concaf 
Need an ACK from PM: @koustavsaha
Need an ACK from Product experience: @RickJWagner 
Need an ACK from QE: @VeereshAradhya  
Need an ACK from DPM: @pmkovar 
Need an ACK from CS: @Preeticp

### CODE REVIEW

**Comment by shipsing** [comment]
@savitaashture @concaf @siamaksade @RickJWagner @VeereshAradhya, Could you please review this PR and share your feedback? Thanks

**Review by shipsing — COMMENTED** [review]
The pipelines 1.9 entry in the matrix table shows the PAC value as 0.15.x (GA), but the page here: http://dashboard.apps.cicd.ospqa.com/releases/componentmatrix/ suggests the PAC value is 0.14.x (GA). So, the published RN and the matrix page need to be consistent. I guess we might need to update either of the two.

Also, Pipelines 1.8 is supported in OCP 4.10. So, manually cherrypicked Pipelines 1.8 RN to OCP 4.10.

File: modules/op-tkn-pipelines-compatibility-support-matrix.adoc [hunk 1]
```diff
@@ -23,7 +23,7 @@ GA:: General Availability
```
Comment (VeereshAradhya) [inline modules/op-tkn-pipelines-compatibility-support-matrix.adoc:23]: `1.11.x` is TP. 
cc: @vdemeester @concaf @ppitonak @PuneetPunamiya

Comment (PuneetPunamiya) [reply modules/op-tkn-pipelines-compatibility-support-matrix.adoc:23]: yes

Comment (shipsing) [reply modules/op-tkn-pipelines-compatibility-support-matrix.adoc:23]: Incorporated this change. Thanks

**Review by savitaashture — COMMENTED** [review]
LGTM

**Review by concaf — APPROVED** [review]

**Review by VeereshAradhya — APPROVED** [review]

**Comment by shipsing** [comment]
@koustavsaha please review and approve the PR. Thanks

**Review by koustavsaha — APPROVED** [review]
ack

**Pull request merged**